styles.css

```
@@ -0,0 +1,359 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;1,300;1,400&family=Oleo+Script+Swash+Caps:wght@400;700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,400&family=Roboto:ital,wght@0,300;0,400;0,500;1,100;1,300&display=swap');
+
+/*Paars #5722CD;
+Blauw #4845E4;
+Roze #ED4385;
+Zwart #0C0C0C (voor standaard tekst);
+Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services") Uitzonderingen:
+Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)
+Grijs #7A7A7A (voor de service-tegel tekst); */
+
+*{
+    margin: 0;
+    padding:0;
+    box-sizing:border-box;
+}
+
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+:root {
+    --default-margin: 16px;
+    --purple:#5722CD;
+    --blue:#4845E4;
+    --lightpurple:#F2F0FF;
+    --lightgrey:#F3F6F9;
+    --grey:#7A7A7A;
+    --font-family: 'Roboto', sans-serif;
+}
+
+h1, h2, h3, h3, h5, h6 {
+    font-family: 'Merriweather', serif;
+   }
+
+body{
+   font-family: var(--font-family);
+}
+
+h1 {
+    font-size: 30px;
+}
+
+h2 {
+    font-size: 1.2em;
+}
+
+
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+.outer-content-container{
+    display:flex;
+    width:100vw;
+    justify-content:center;
+}
+
+.inner-content-container{
+    display:flex;
+    flex-direction:column;
+    max-width: 1400px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    margin: 0 20px;
+    align-items:center;
+}
+
+header{
+    background: rgb(72,69,228);
+    background: linear-gradient(90deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 100%);
+    width:100%;
+    position:relative;
+    height:500px;
+    padding:0px 0px 80px 0px;
```

Als drie waarden 0px zijn en ze niet een andere waarde overschrijven, dan zie ik persoonlijk liever
```suggestion
    padding-bottom: 80px;
```